server/rpc.go

```diff
@@ -367,9 +367,29 @@ func getOrchestrator(orch Orchestrator, req *net.OrchestratorRequest) (*net.Orch
 		return orchestratorInfo(orch, addr, orch.ServiceURI().String(), "")
 	}
 
+	if err := checkLiveVideoToVideoCapacity(orch, req.Capabilities); err != nil {
+		return nil, fmt.Errorf("Invalid orchestrator request: %v", err)
+	}
 	return orchestratorInfoWithCaps(orch, addr, orch.ServiceURI().String(), "", req.Capabilities)
 }
 
+func checkLiveVideoToVideoCapacity(orch Orchestrator, caps *net.Capabilities) interface{} {
+	if caps.Constraints != nil && caps.Constraints.PerCapability != nil {
```

Comment:
nit: might be nice to reverse some of these if statements and return early to reduce the indentation

Reply:
Good idea. Updated.